Packs/ShiftLeft/Integrations/shiftleft/shiftleft.py

```diff
@@ -0,0 +1,338 @@
+            outputs=apps,
+        )
+    else:
+        return CommandResults(
+            outputs_prefix="ShiftLeft.Apps",
+            outputs_key_field="",
+            outputs={},
+        )
+
+
+def list_app_secrets_command(
+    client: ShiftLeftClient, org_id: str, args: Dict[str, Any]
+) -> CommandResults:
+    app_name = args.get("app_name", None)
+    if not app_name:
+        raise ValueError("app_name not specified")
+    version = args.get("version", None)
+    entropy = args.get("entropy", 0.48)
+    result = client.list_app_findings(org_id, app_name, None, "secret", version)
+    response: Any = result.get("response") if result.get("ok") else {}
+    findings = response.get("findings", {})
+    filtered_findings = [
+        f
+        for f in findings
+        if float(f.get("details", {}).get("entropy", 0)) > float(entropy)
+    ]
+    for f in filtered_findings:
+        f["secret"] = ""
+        f["fileName"] = ""
+        details = f.get("details")
+        if details:
+            f["secret"] = details.get("secret", "")
+            f["fileName"] = details.get("fileName", "")
+            if details.get("vcsUrl"):
+                f[
+                    "fileName"
+                ] = f'[{details.get("fileName", "Location")}]({details.get("vcsUrl")})'
+    scan = response.get("scan", [])
+    if scan:
+        scan = [scan]
+    if response:
+        markdown = "### Scan Info\n"
+        markdown += tableToMarkdown(
+            "",
+            scan,
+            headers=[
+                "id",
+                "app",
+                "language",
+                "version",
+                "started_at",
+                "completed_at",
+                "number_of_expressions",
+            ],
+        )
+        markdown += "\n### Recommendation\n"
+        if len(filtered_findings):
+            markdown += "\nReview the secrets in this list and follow the appropriate Secrets Disclosure & Management playbook!"
+        else:
+            markdown += "\nNo secrets in this app require your attention."
+        markdown += f"\n### ShiftLeft Findings ({len(filtered_findings)})\n"
+        markdown += tableToMarkdown(
+            "",
+            filtered_findings,
+            headers=[
+                "id",
+                "title",
+                "secret",
+                "fileName",
+            ],
+        )
+        return CommandResults(
+            readable_output=markdown,
+            outputs_prefix="ShiftLeft.Secrets",
+            outputs_key_field="scan.id",
+            outputs=response,
+        )
+    else:
+        return CommandResults(
+            outputs_prefix="ShiftLeft.Secrets",
+            outputs_key_field="",
+            outputs={},
+        )
+
+
+def list_app_findings_command(
+    client: ShiftLeftClient, org_id: str, args: Dict[str, Any]
+) -> CommandResults:
+    app_name = args.get("app_name", None)
+    if not app_name:
+        raise ValueError("app_name not specified")
```

Comment:
It is a mandatory arg so this condition is useless. However, you can still keep it.
```suggestion
    app_name = args.get("app_name")
    if not app_name:
        raise ValueError("Shiftleft error: app_name not specified.")
```